src/App.vue

```diff
@@ -62,7 +62,12 @@ export default {
       this.teams[index].score++
     },
     subtractPoint(index) {
```

Comment:
Can we get rid of everything except the below lines within the subtractPoint method?

```suggestion
    if (this.teams[index].score > 0) {
        this.teams[index].score--
      }
```

Reply:
Ok so suggested code changes are new in GitHub, so I'm still getting used to how to best use them. The above suggestion didn't mean to delete line 64 😅 just have a single if statement and get rid of the else statement entirely